### PR TITLE
Docs: Field config - add __field.labels syntax to Display name section

### PR DESCRIPTION
**Pull request by kylebrandt**
Documents Display name Field configuration expression that is missing from current docs.

relates to #26470

### CODE REVIEW

File: docs/sources/panels/field-configuration-options.md [hunk 1]
```diff
@@ -139,7 +139,9 @@ For more information and instructions, refer to [Data links]({{< relref "../link
 
 Lets you set the display title of all fields. You can use [variables]({{< relref "../variables/templates-and-variables.md" >}}) in the field title.
 
-When multiple stats are shown, this field controls the title in each stat. By default this is the series name and field name. You can use expressions like ${__series.name} or ${**field.name} to use only series name or field name in title or \${**cell_2} to refer to other fields (2 being field/column with index 2).
+When multiple stats are shown, this field controls the title in each stat. By default this is the series name and field name. You can use expressions like `${__series.name}` or `${**field.name}` to use only series name or field name in title or `${**cell_2}` to refer to other fields (2 being field/column with index 2).
```
Comment (ryantxu) [inline docs/sources/panels/field-configuration-options.md:142]: what is `${**field...`??  Is it a typo?

**Review by ryantxu — APPROVED** [review]
addition of labels looks good.  The `**` seems like an error though

File: docs/sources/panels/field-configuration-options.md (hunk 1 above)
Comment (kylebrandt) [reply docs/sources/panels/field-configuration-options.md:142]: I thought maybe it was an alternate syntax? :P Will change.

**Comment by kylebrandt** [comment]
> addition of labels looks good. The `**` seems like an error though

Updated. `${__cell_1} doesn't seem to work, also the name doesn't make much sense to me. My understanding is that a cell would be identifed by a column in addition to a row idx. So if this is a thing that works somewhere, perhaps it should be __col or __column ? Does this cell syntax work anywhere?

**Comment by kylebrandt** [comment]
@ryantxu Also related - debate of what `field.name` should resolve to: https://github.com/grafana/grafana/issues/26370

**Review by oddlittlebird — APPROVED** [review]
Looks good except for one minor edit.

**Comment by kylebrandt** [comment]
closing in favor of https://github.com/grafana/grafana/pull/26489

**Pull request closed without merging**